static/bidder-info/pubmatic.yaml

```diff
@@ -11,7 +11,11 @@ capabilities:
       - banner
       - video
 userSync:
+  default: "iframe"
```

Comment:
We have since removed the `default` field and have hardcoded to `iframe` based on bidder feedback. Please delete this line.

Reply:
done